content/departments/cloud/technical-docs/private-code-hosts/index.md

```diff
@@ -0,0 +1,153 @@
+# Managed Instance Private Code Hosts support
+
+> **Warning**
+> This is still work in progress!
```

Comment:
nit - you can get nice colourful annotation with:

```suggestion
> WARNING: This is still work in progress!
```

Reply:
Thx!